setup.py

```diff
@@ -23,16 +23,14 @@
 AUTHOR_EMAIL = "rflperry@gmail.com"
 URL = "https://github.com/mvlearn/mvlearn"
 MINIMUM_PYTHON_VERSION = 3, 6  # Minimum of Python 3.6
-with open("./requirements/base.txt", "r") as f:
-    REQUIRED_PACKAGES = f.read()
-with open("./requirements/torch.txt", "r") as f:
-    torch_extras = f.read()
-with open("./requirements/multiviewica.txt", "r") as f:
-    multiviewica_extras = f.read()
-EXTRA_PACKAGES = {
-    'torch': torch_extras,
-    'multiviewica': multiviewica_extras,
-}
+REQUIRED_PACKAGES = [
+    "matplotlib>=3.0.0, <=3.3.4",
```

Comment:
can we avoid the <=3.3.4? I don't like pinning versions in libraries or limiting max version and it leads to env that cannot be updated

Reply:
I believe this was changed because some of the examples notebooks broke for 3.4 and above. There is nothing in mvlearn itself that requires the maximum, so it should be fine to remove the <=3.3.4 from here and then work on updating the example notebooks as a follow-up